pageserver/src/page_service.rs

```diff
@@ -477,6 +478,13 @@ impl Connection {
                 self.stream.write_i32::<BE>(0)?;
             }
 
+            BeMessage::ParameterStatus => {
+                self.stream.write_u8(b'S')?;
+                const PARAM: &[u8] = b"client_encoding\0UTF8\0";
+                self.stream.write_i32::<BE>(4 + PARAM.len() as i32)?;
+                self.stream.write(PARAM)?;
+            }
```

Comment:
The bit seems kind of mysterious: what does this magic string do? What is the significance of `4 + PARAM.len()`?

Perhaps a comment here to explain?